packages/react/src/AvatarStack/AvatarStack.module.css

```diff
@@ -47,26 +45,18 @@
     /*
     MIN-WIDTH CALC FORMULA EXPLAINED:
     avatar size ➡️ var(--avatar-stack-size)
-    plus the visible part of the 2nd avatar ➡️ var(--avatar-stack-size) - var(--overlap-size)
-    plus the visible part of the 3rd avatar ➡️ var(--avatar-stack-size) - var(--overlap-size-avatar-three-plus)
+    plus the visible part of the 2nd and 3rd avatars ➡️ (var(--avatar-stack-size) - var(--overlap-size)) * 2
     */
-    min-width: calc(
-      var(--avatar-stack-size) + (var(--avatar-stack-size) - var(--overlap-size)) +
-        (var(--avatar-stack-size) - var(--overlap-size-avatar-three-plus))
-    );
+    min-width: calc(var(--avatar-stack-size) + ((var(--avatar-stack-size) - var(--overlap-size)) * 2));
   }
 
   &:where([data-avatar-count='3+']) {
     /*
     MIN-WIDTH CALC FORMULA EXPLAINED:
     avatar size ➡️ var(--avatar-stack-size)
-    plus the visible part of the 2nd avatar ➡️ var(--avatar-stack-size) - var(--overlap-size)
-    plus the visible part of the 3rd AND 4th avatar ➡️ (var(--avatar-stack-size) - var(--overlap-size-avatar-three-plus)) * 2
```

Comment:
Currently, the max amount of avatars that can be displayed at rest is 5 - shouldn't the `min-width` include the 5th avatar?